index.html

```diff
@@ -22,17 +22,15 @@
 
       <nav class="nav-bar">
         <div class="left-navbar">
-          <a href="#" class="logo">Muhammad Zunair khan</a>
-          <div class="wrap">
-            <a href="#">
-              <div class="menu"></div>
-              <div class="menu"></div>
-              <div class="menu"></div>
-            </a>
+          <a id="logo" href="#" class="logo">Muhammad Zunair khan</a>
+          <div id="wrap" class="wrap" onclick="onClickMenu()">
```

Comment:
- [ ] The mobile menu should take the whole height of the page when open, kindly consider implementing this so your site looks identical to the Figma design
- [ ] Also kindly make sure the font wight and the close button color is similar to the Figma design

|Your website|Figma design|
|-|-|
|![Screenshot_20230130_210351](https://user-images.githubusercontent.com/25227022/215583551-ad4cba6f-dab1-4eef-bc23-82babb0d009e.png)|![Screenshot_20230130_210552](https://user-images.githubusercontent.com/25227022/215583505-1695f761-965a-4e53-97d7-650e3b63eca0.png)|